src/fns/constrained_ops.nr

```diff
@@ -35,6 +36,24 @@ use crate::fns::{
 * We use a hash function that can be modelled as a random oracle
 * This function *should* produce an output that is a uniformly randomly distributed value modulo BigNum::modulus()
 **/
+pub(crate) fn from_field<let N: u32, let MOD_BITS: u32>(
+    params: P<N, MOD_BITS>,
+    field: Field,
+) -> [Field; N] {
+    let result = __from_field::<N>(field);
+    // validate the limbs are in range and the value in total is less than 2^254
+    validate_in_range::<N, 254>(result);
```

Comment:
I wonder if there's a bug here. it might be that the result is 254 bits long but larger than the field modulus. 
Should I create a field params and do a `validate_in_field`?